config.example.yaml

```diff
@@ -14,6 +14,11 @@ aws:
     access-key: "ASD7ASFET65FFGHDFFS"
     secret-access-key: "fjhgsadf6#$!@%&/dfghdfgdsdf"
     account-id: "123434565665"
+elastic:
```

Comment:
Is it possible to take this in as a single line url that already includes the user/pass like we do with benchmark-operator/wrapper?

Reply:
It works also with the URL with user/password included. You can leave omit the user and password configs.